NOTICE

```diff
@@ -0,0 +1,20 @@
+Apache Daffodil
+Copyright 2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+
+******************
+W3C Software License
+******************
+
+  This product includes material copied or derived from W3C, Copyright 2001 (c)
+  World Wide Web Consortium (Massachusetts Institute of Technology, European
+  Research Consortium for Informatics and Mathematics, Keio University). Specific
+  files include:
+    - XMLSchema.dtd (https://www.w3.org/2001/XMLSchema.dtd)
+    - XMLSchema.xsd (https://www.w3.org/2001/XMLSchema.xsd)
+    - xml.xsd (https://www.w3.org/2001/xml.xsd)
+    - datatypes.dtd (https://www.w3.org/2001/datatypes.dtd)
+    - XMLSchema_for_DFDL.xsd (https://www.w3.org/2001/XMLSchema.dtd)
```

Comment:
A discussion recently popped up on the Apache incubator list about what belongs in the NOTICE file:

https://lists.apache.org/thread.html/911ba67907cbca6925851e448ac8a7e73a921cb59efb8ca028e15845@%3Clegal-discuss.apache.org%3E

This specifically mentions the W3C license. As I understand it, the conclusion is that we should be careful what goes in a NOTICE file since it is sortof viral. And since the W3C License does not specifically call out how attribute should be made (like the Apache v2 license), it is not required to be put in the NOTICE file. The same content is already in the LICENSE file which is sufficient for attribution, so this should be removed. Same with the content in daffodil-cli/NOTICE.